contrib/spire-lookup.ps1

```diff
@@ -0,0 +1,21 @@
+$ErrorActionPreference = "Stop"
+
+$inFile = Get-ChildItem $args[0]
+$outFile = New-Item -ItemType File -Path "$($inFile.BaseName)-result$($inFile.Extension)"
+
+foreach($row in (Import-Csv -Path $inFile)) {
+  $userLookup = Get-ADUser -Filter "employeeID -eq $($row.spire_id) -and employeeType -like 'P*'" `
```

Comment:
`Get-ADUser` is part of the Active Directory PowerShell module it comes with [RSAT](https://learn.microsoft.com/en-US/troubleshoot/windows-server/system-management-components/remote-server-administration-tools).